src/test/java/be/syntra/devshop/DevshopFront/services/utils/ProductMapperTest.java

```diff
@@ -58,15 +58,12 @@ void convertToProductsDisplayListDto() {
         // given
         final ProductList dummyProductList = getDummyProductList();
         final CartDto dummyCartDto = getCartWithOneDummyProduct();
-        when(cartService.getCart()).thenReturn(dummyCartDto);
```

Comment:
remove the @Mock CartService, and there's an dummyCartDto that's also unused now

Reply:
fixed